extra/stoatLordCardStuff.js

```diff
@@ -31,6 +31,33 @@ async function load() {
         cardFormated.attack = card["Power"]
         cardFormated.health = card["Health"]
 
+        // parsing cost
+        card["Cost"] = card["Cost"]
+            .replace("bones", "bone")
+            .replace("sapphires", "sapphire")
+            .replace("rubies", "ruby")
+            .replace("prisms", "prism")
+        for (let cost of card["Cost"].split(",")) {
+            cost = cost.trim().toLowerCase()
+            let temp = cost.split(" ")
+            if (cost.includes("shattered")) {
+                if (!cardFormated["shattered"]) cardFormated["shattered"] = []
+                for (let i = 0; i < temp[0]; i++) {
+                    cardFormated["shattered"].push(`shattered_${temp[2]}`)
+                }
+            } else if (["sapphire", "ruby", "emerald", "prism"].some((i) => cost.includes(i))) {
+                if (!cardFormated["gem"]) cardFormated["gem"] = []
+                for (let i = 0; i < temp[0]; i++) {
+                    cardFormated["gem"].push(temp[1])
```

Comment:
It should be `mox` not `gem` refer to https://github.com/Mouthless-Stoat/MagpieTutor/blob/master/index.js#L246